src/bin/archive-storybook.ts

```diff
@@ -3,6 +3,9 @@
 import { execFileSync } from 'child_process';
 import { resolve, dirname } from 'path';
 
+// Discard first two entries (exec path and file path)
+const args = process.argv.slice(2);
+
 const configDir = 'node_modules/@chromaui/archive-storybook/config';
 const binPath = resolve(dirname(require.resolve('@storybook/cli/package.json')), './bin/index.js');
-execFileSync(binPath, ['dev', '-c', configDir], { stdio: 'inherit' });
+execFileSync(binPath, ['dev', ...args, '-c', configDir], { stdio: 'inherit' });
```

Comment:
This will ensure that our `configDir` wins over one that is included in the args. Is that what we want?

Reply:
@tevanoff I believe we want the supplied (`args`) config directory to win out, since this will allow people to do things like `build-archive-storybook -c some-dir` which they currently can't (they currently have to use the `build-storybook` script to pass a custom config directory). Do you see any issues with that?

Reply:
@skitterm I just don't know if that's something we want to support at this point. The Storybook used for this is hidden, and its config file is set up in a specific way for these e2e tests. If that can be overridden and changed, it introduces more chances for errors. I think that's maybe not a door we want to open up at this early stage.

Reply:
@tevanoff currently customers have to indicate the config directory if in a hoisted monorepo (tests and this command are in a subdirectory, all node_modules (including our storybook config) are at the root). Currently they have to use `build-storybook` to get that logic. We're looking into being able to resolve the storybook config directory internally though, so that'd remove the need for the user to specify.

I guess the larger question is -- should we allow all storybook arguments to pass through to `archive-storybook`? I would assume we would to avoid confusion of why some things work and some don't, but I guess it depends on how many of the options seem useful for end-to-end.

Reply:
I'm not sure there's a heap of point in calling `archive-storybook` (instead of just `storybook`) if you are going to provide the `-c` option though.

Reply:
@skitterm and I chatted a bit, we'll leave this as is for now and look into finding the correct config path in a monorepo setup to obviate the need for overriding it.